docs/mobile-apps/features/audio-capture.md

```diff
@@ -0,0 +1,81 @@
+---
+id: audio-capture
+title: Audio Capture
+---
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+# Audio Capture 
+
+Audio Capture is a functionality that gives you the ability to record the audio stream generated by your native mobile app (Android or iOS/iPadOS) during a live or automated real device test. 
+
+With this feature, the audio is recorded on our side and merged with the video recording, which you can access in your **Test Results**.
+
+## What You'll Need
+
+* A Sauce Labs account ([Log in](https://accounts.saucelabs.com/am/XUI/#login/) or sign up for a [free trial license](https://saucelabs.com/sign-up))
+* A native Android, iOS, or iPadOS mobile app that makes HTTP/HTTPS requests. We currently support the following:
+  * Android 10 and higher
+  * iOS 10+ up to iOS 15.3
+
+
+## Using Audio Capture on Real Devices
+
+To enable audio capture in your tests: 
+ 
+1. On Sauce Labs, click **Live** > **Mobile App**.
+2. [Upload your mobile app to Sauce Labs](/mobile-apps/app-storage) through the UI or CLI.
+3. After you’ve uploaded your app, return to the **Live** > **Mobile App** page, hover your mouse over your app, then select **Settings**. <br/><img src={useBaseUrl('img/mobile-apps/networkcapturescr.png')} alt="Mobile app settings navigation" width="800"/>
+4. Under Default Settings, toggle Instrumentation and App/OS Audio to enable the feature. <br/><img src={useBaseUrl('img/mobile-apps/audio_capture_screen2.png')} alt="Mobile app settings navigation" width="800"/>
+5. **For Automated Testing only:** add the `audioCapture` capability to your test script. Click the link below corresponding to your framework:
+   * Espresso (coming soon)
+   * XCUITest (coming soon)
+   * [Appium](/dev/test-configuration-options/#audiocapture)
+  
+  
+Now you can start your live or automated testing session.Your audio recording will be captured and be part of the video recording in the test results page.
+
+## Accessing the Audio in Your Test Tesults
+
+The audio recording will be included into the video recording, which you can play back and download from the built-in media player on the **Test Results** page. 
+
+You can also download the video file with the included audio stream programmatically using the following API request: 
+
+```
+curl --compressed \ -O https://{SAUCE_USERNAME}:{SAUCE_ACCESS_KEY}@{DATA_CENTER}.saucelabs.com/v1/rdc/jobs/{JOB_ID}/video.mp4
+```
```

Comment:
@aoleszyn I would move Test Results section after the Android/iOS/iPadOS instructions to align with the user workflow (i.e., enable > run test > view test results).

Reply:
Sections rearranged accordingly.